runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -43,24 +43,7 @@
 import net.runelite.api.Tile;
 import net.runelite.api.TileObject;
 import net.runelite.api.coords.WorldPoint;
-import net.runelite.api.events.ConfigChanged;
```

Comment:
You need to fix your imports according to the wiki

https://github.com/runelite/runelite/wiki/Code-Conventions